### ISSUE TITLE
English version

### ISSUE THREAD

**Issue by rece225** [togenyan/Yo-kai_Editor_3#31]
Please update and post a version of the program in english is so hard at least for me even if I can understand just a little bit of japanese use all the program features, thanks :D and i'll wait for it in sukiyaki I LOVE what you're doing here, thanks again. :'J

**Comment by togenyan** [comment]
Could you please test the latest version I released today?
https://github.com/togenyan/Yo-kai_Editor_3/releases/tag/v0.6.4

**Comment by rece225** [comment]
yes, I'll test it but I mean, all the yo-kai names and items weren't in english in the latest version before this, and those names are the ones I need, so I'll see what's new in this version, thanks!.
